src/App.js

```diff
@@ -3,18 +3,23 @@ import './App.css';
 
 import TimerContext from './TimerContext'
 import Nav from './components/nav'
-import Canvas from './components/canvas'
+import Canvas from './components/canvas/'
```

Comment:
You don't need this last `/`. 🤔

```suggestion
import Canvas from './components/canvas'
```